src/mds/MDSDaemon.cc

```diff
@@ -64,7 +64,6 @@
 MDSDaemon::MDSDaemon(std::string_view n, Messenger *m, MonClient *mc) :
   Dispatcher(m->cct),
   mds_lock(ceph::make_mutex("MDSDaemon::mds_lock")),
```

Comment:
This can be moved into the header too:

`... mds_lock = ceph::make_mutex("...");`

Reply:
Done!